geomesa-utils/src/main/scala/org/locationtech/geomesa/utils/cache/CacheKeyGenerator.scala

```diff
@@ -0,0 +1,19 @@
+/***********************************************************************
+* Copyright (c) 2013-2016 Commonwealth Computer Research, Inc.
+* All rights reserved. This program and the accompanying materials
+* are made available under the terms of the Apache License, Version 2.0
+* which accompanies this distribution and is available at
+* http://www.opensource.org/licenses/apache2.0.php.
+*************************************************************************/
+
+package org.locationtech.geomesa.utils.cache
+
+import org.opengis.feature.simple.SimpleFeatureType
+
+object CacheKeyGenerator {
+
+  import collection.JavaConversions._
+
+  def cacheKey(sft: SimpleFeatureType): String =
```

Comment:
I kind of dislike package objects because the imports are messy... I find it a lot clearer to find/import a class than try to remember which package object things come from.